en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -72,10 +77,82 @@ The image below shows the wiring in greater detail.
 
 ![Image showing writing from the PM03D power module to the baseboard](../../assets/companion_computer/holybro_pixhawk_rpi_cm4_baseboard/baseboard_wiring_guide.jpg)
 
+# RPi CM4 Flash Guide
+
+### Please **Note:**
+
+* If you are using PX4, you will need to use PX4 version 1.13.1 or newer for PX4 to recognize this baseboard.
+* The fan does not indicate if the RPi CM4 is powered/running or not.
+* The power module plugged into Power1/2 does not power the RPi part. You can use the additional USB-C Cable from the PM03D power module to the CM4 Slave USB-C port.
+* The Micro-HDMI port is an output port.
+* Some RPi CM4 might not have Wifi device and therefore won’t connect automatically, unless you plug it into a router or a compatible Wifi dongle into the CM4 Host ports.
+
+### Flash EMMC <a href="#flash-emmc-2" id="flash-emmc-2"></a>
+
+We need to flash a RPi image onto EMMC.
+
+1. Switch Dip-Switch to RPI.
+2. Connect computer to USB-C _CM4 Slave_ port used power & flash the RPi.
+3. Get usbboot, build it and run it.
+
+```
+sudo apt install libusb-1.0-0-dev
+git clone --depth=1 https://github.com/raspberrypi/usbboot,
+cd usbboot
+make
+sudo ./rpiboot
+```
+
+* You can now install your favorite Linux distro, e.g. Raspberry Pi OS 64bit, using The `rpi-imager`. Make sure to add wifi and ssh settings (hidden behind the gear/advanced symbol).
+
+```
+sudo apt install rpi-imager
+rpi-imager
+```
+
+1. Once done, unmount the volumes, and power off the CM4 by unplugging USB-C CM4 Slave.
+2. Switch Dip-Switch back to EMMC.
+3. Power on CM4 by providing power to USB-C CM4 Slave port.
+4. To check if it’s booting/working, either check HDMI output, or connect via ssh (if set up in rpi-imager, and wifi is available).
+
+### Connect PX4 to CM4 via serial <a href="#connect-px4-to-cm4-via-serial-3" id="connect-px4-to-cm4-via-serial-3"></a>
+
+Pixhawk 6X talks to CM4 using Telem2 (`/dev/ttyS4`).
+
+1. To enable this MAVLink instance, set the params:\
```

Comment:
FYI backslashes are never needed except in the rare case you want to escape a markdown symbol